controllers/depresolver/depresolver_spec.go

```diff
@@ -24,15 +25,21 @@ func (dr *DependencyResolver) ResolveGslbSpec(ctx context.Context, gslb *k8gbv1b
 		if strategy.SplitBrainThresholdSeconds == 0 {
 			strategy.SplitBrainThresholdSeconds = predefinedStrategy.SplitBrainThresholdSeconds
 		}
-		dr.errorSpec = dr.validateSpec(strategy)
+		dr.errorSpec = dr.validateStrategy(strategy)
```

Comment:
@kuritka , what was the initial intention of this method, spec validation in general or just gslb strategy parameters validation?